crates/router/src/core/api_keys.rs

```diff
@@ -270,6 +270,14 @@ pub async fn add_api_key_expiry_task(
                 api_key_expiry_tracker.key_id
             )
         })?;
+    metrics::TASKS_ADDED_COUNT.add(
+        &metrics::CONTEXT,
+        1,
+        &[metrics::request::add_attributes(
+            "api_key_expiry",
```

Comment:
Please keep the key the same everywhere `TASKS_ADDED_COUNT` and `TASKS_RESET_COUNT` are being incremented. That way filtering becomes easier:

If you'd use `flow` for the key, the attributes would be `flow = api_key_expiry`, `flow = delete_tokenize_data`, `flow = payments_confirm`, etc. as compared to `api_key_expiry = 7` and `delete_tokenize_data = DELETE_TOKENIZE_DATA`, which might not make much sense.